rflx/pyrflx/message.py

```diff
@@ -103,8 +103,10 @@ def _prev_field(self, fld: str) -> str:
 
     def _get_length_unchecked(self, fld: str) -> Expr:
         for l in self._model.incoming(model.Field(fld)):
+            # genutzte Kante -> True und die Länge explizit an der genutzten Kante angegeben
```

Comment:
Please use only english comments. Also check if the comment is necessary to be in the commit. If comments are only temporarily, omit them when committing in the future.